packages/rsocket-flowable/src/FlowableMapOperator.js

```diff
@@ -49,7 +47,10 @@ export default class FlowableMapOperator<T, R> implements ISubscriber<T> {
     try {
       this._subscriber.onNext(this._fn(t));
     } catch (e) {
-      nullthrows(this._subscription).cancel();
+      if (!this._subscription) {
+        throw new Error('subscription is null');
```

Comment:
Very minor detail but this changes the exception message now. Not sure we care though?